zeusd/src/devices/cpu/linux.rs

```diff
@@ -149,7 +158,7 @@ impl CpuManager for RaplCpu {
         Ok(measurement + num_wraparounds * self.cpu.max_energy_uj)
     }
 
-    fn get_dram_energy(&self) -> Result<u64, ZeusdError> {
+    fn get_dram_energy(&mut self) -> Result<u64, ZeusdError> {
```

Comment:
I had to add it for the `TestCpu` because it implements the `CpuManager` trait in test/mod.rs

```
    fn get_cpu_energy(&mut self) -> Result<u64, ZeusdError> {
        Ok(std::iter::from_fn(|| self.cpu.try_recv().ok())
            .next()
            .unwrap())
    }

```